### DISCUSSION TITLE
Manage SSO users and notifications

### DISCUSSION THREAD

**Discussion by rn232** [firefly-iii/firefly-iii#9369] [Firefly III]
Hello,

I have 2 small questions about installing Firefly.

1. How do I delete a user created by SSO? From the panel it tells us that it's impossible, and at DB level I'm not sure how to go about deleting the user and the elements (transactions, accounts, etc.) linked to him.

2. I use Authentik as my SSO provider. I use the account UID as the user ID (X-authentik-uid), and X-authentik-email for the user's email. Except that in the settings, in email, I have the UID that appears and not the email address. The e-mail address is entered in DB in the preferences table (remote_guard_alt_email). Do the notifications still work? Won't this cause problems over time? Is this a common practice?

Thank you in advance for your answers.
The software looks really great, I can't wait to start using it!

**Comment by JC5** [comment]
Hi, thanks for opening a discussion!

1. Delete the entry from the "users" table. All other data from that user will be deleted automatically.
2. Yes this works.

**Reply by rn232** [reply]
Thank you for you answer.

I tried, and it work as expected. I also deleted him in the user_groups table, just to be sure.
